R/utils-tags.R

```diff
@@ -2,7 +2,7 @@ tag_require <- function(tag, version = 4, caller = "") {
   tagAddRenderHook(
     tag, replace = FALSE,
     func = function(x) {
-      current_version <- theme_version(bs_current_theme()) %||% 3
+      current_version <- theme_version(bs_current_theme()) %||% version_default()
```

Comment:
Why was this needed? Seems like this could give wrong results if this gets called and `bs_theme()` isn't being used for the Bootstrap dependency